docs/data-management.rst

```diff
@@ -280,38 +426,80 @@ Step-by-Step
 
 #. On your local machine, open a terminal
 
-#. Activate the virtual environment where ``omero-py`` is installed or add it to ``PATH`` e.g.::
-    
-   $ export PATH=/opt/omero/server/venv3/bin:$PATH
+#. Activate the virtual environment as indicated in the Setup section above.
 
-#. The variables ``$ID​1`` and ``$ID2`` below are the IDs of the ​selected Datasets. To duplicate two Datasets with their Images and annotations on both the Images and the Datasets, run::
+#. The variables ``$ID​1`` and ``$ID2`` below are the 
+   IDs of the ​selected Datasets. To duplicate two Datasets
+   with their Images and annotations on both the 
+   Images and the Datasets, run::
     
    $ ​omero duplicate Dataset:$ID1,$ID2 --report
 
-#. The duplicated Datasets will not be linked to any Project, even if the originals were linked to some Project. 
+#. The duplicated Datasets will not be linked to any Project,
+   even if the originals were linked to some Project. 
 
-#. Duplicate two Images with many ROIs on them. The ROIs duplication might take a long time. To exclude the duplication of the ROIs, run::
+#. Duplicate two Images with many ROIs on them. 
+   The ROIs duplication might take a long time.
+   To exclude the duplication of the ROIs, run::
 
    $ omero duplicate Image:$ID1,$ID2 --ignore Roi --report
 
-#. Find the duplicated Images in the Orphaned Images and Drag and Drop them into a Dataset or create a new Dataset for them.
-
-#. Duplicate two Projects of another user in read-annotate group type. The group name in our example below is `read-annotate-group`. Log in as `user-1` and duplicate Projects belonging to `user-2` in that group. Further, instead of duplicating all the annotations of `user-2`, link all the original annotations such as Tags, Key-Value pairs and FileAnnotations of `user-2` to the corresponding objects in the duplicate, and duplicate only Comments and Ratings. The duplicate and Comments and Ratings on it will belong to `user-1`, but the other annotations will still belong to `user-2`. Run::
+#. Find the duplicated Images in the Orphaned Images
+   and Drag and Drop them into a Dataset or create a new Dataset for them.
+
+#. Duplicate two Projects of another user in read-annotate group type.
+   This will duplicate also the Datasets linked to those Projects
+   as welll as Images linked to those Datasets. 
+   Here we specify some reference classes of objects that
```

Comment:
What does `reference classes` mean?

Reply:
> What does reference classes mean?

That is explained right below that sentence. The class of objects such as "TagAnnotation" will become a ``reference`` class by virtue of being explicitly flagged so in the duplicate command. Being a reference class in this sense means that the objects will not be duplicated, but referenced instead, i.e. the originals of the "reference" objects will be linked to the duplicates.

Reply:
@manics Could you attempt a reformulation which would capture the meaning better please ?

Reply:
> @manics Could you attempt a reformulation which would capture the meaning better please ?

Actually, I think I found an acceptable reformulation in https://github.com/ome/omero-guide-introduction/pull/17/commits/ed206030ddac9449e661915a68b396d1d221efa1